python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -721,11 +715,6 @@ def get_edge_data(self, edge_ids=None, types=None, columns=None):
         if self.__edge_prop_dataframe is not None:
             df = self.__edge_prop_dataframe
             if edge_ids is not None:
-                if isinstance(edge_ids, int):
-                    edge_ids = [edge_ids]
-                elif not isinstance(edge_ids,
-                                    (list, slice, self.__series_type)):
-                    edge_ids = list(edge_ids)
```

Comment:
I think we should let the  underlying library handle it rather than limiting `location` to these types. 

Also, this fails with GPU objects like `cupy` , `cudf.Series` , for numpy arrays we pay a cost.

Reply:
Fails when fetching a single element.